sql/index/config.go

```diff
@@ -0,0 +1,107 @@
+package index
```

Comment:
Should we add some documentation to methods and constructors on this file?

Reply:
Added extra comments. So far I think it's clear what it does.